skywalking-collector/skywalking-collector-cluster/src/main/java/com/a/eye/skywalking/collector/AkkaSystem.java

```diff
@@ -18,8 +18,11 @@
 
     public ActorSystem create() {
         final Config config = ConfigFactory.parseString("akka.remote.netty.tcp.HOSTNAME=" + ClusterConfig.Cluster.Current.HOSTNAME).
-            withFallback(ConfigFactory.parseString("akka.remote.netty.tcp.PORT=" + ClusterConfig.Cluster.Current.PORT)).
-            withFallback(ConfigFactory.load("application.conf"));
+                withFallback(ConfigFactory.parseString("akka.remote.netty.tcp.PORT=" + ClusterConfig.Cluster.Current.PORT)).
+                withFallback(ConfigFactory.parseString("akka.loggers=[\"akka.event.slf4j.Slf4jLogger\"]")).
+                withFallback(ConfigFactory.parseString("akka.loglevel=\"ERROR\"")).
```

Comment:
I suggest this `loglevel` config should be initialized by `collector.config`. The config key may be `cluster.akka.loglevel`?